week1/assignment/todo.rb

```diff
@@ -78,13 +89,25 @@ def prompt
     end
   end
 
+  def save
+    File.open("todo.md", "w") do |f|
+      @list.items.map do |item|
```

Comment:
you're using `.map` here but inside the block you're not returning anything, so this is more appropriate as `.each`